client/src/telemetry/glean-context.tsx

```diff
@@ -146,6 +163,15 @@ export function useGleanPage() {
         userAgent: navigator?.userAgent,
         geo: userData?.geo?.country,
         subscriptionType: userData?.subscriptionType || "anonymous",
+        viewportBreakpoint: BREAKPOINTS.find(
+          ([_, width]) => width <= window.innerWidth
+        )?.[0],
+        viewportRatio: Math.round(
+          (100 * window.innerWidth) / window.innerHeight
+        ),
```

Comment:
I find it confusing to have the ratio as a percentage, and I wouldn't think we need so much precision (as the values go above 1 / 100% anyways, e.g. 3.28 / 328% for me):

```suggestion
        viewportRatio: Math.round(
          (10 * window.innerWidth) / window.innerHeight
        ) / 10,
```

Reply:
As discussed on slack: glean can't take a float, so keeping as an int in percentage rather than a string in decimal